integreatly/integreatly-1.7.6/integreatly-operator.1.7.6.clusterserviceversion.yaml

```diff
@@ -0,0 +1,366 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: integreatlyoperator.1.7.6
```

Comment:
I think the version in the integreatly-operator would also require a bump, looks like the version there is currently `1.12.0`

Reply:
as not including it doesn't matter